### DISCUSSION TITLE
Inquiry on PVNet and CloudCast

### DISCUSSION THREAD

**Discussion by kwon-encored** [openclimatefix/community#14] [Help] [answered]
@dfulu 
Hi James Fulton,  

I hope this message finds you well. Over the past few months, I have been exploring your work on the Open Climate Fix model and research regarding the *Multimodal Late-Fusion Network* for forecasting regional PV power. I found your [paper](https://www.climatechange.ai/papers/iclr2024/46) and [GitHub repository](https://github.com/openclimatefix/PVNet) to be incredibly insightful.  

While researching multimodal techniques, particularly those utilizing satellite imagery, I came across another study by *The Alan Turing Institute* titled [Quartz Solar AI Nowcasting](https://www.turing.ac.uk/research/research-projects/quartz-solar-ai-nowcasting), where I noticed your involvement.  

I’m eager to delve deeper into this study and understand how its approach (e.g., CloudCast) compares to Open Climate Fix's PVNet. Would it be possible for you to share, if any, **published papers**, documents, resources, or insights related to these projects?  

Thank you for your time and consideration. I look forward to learning more from your expertise!

**Comment by dfulu** [comment, marked as answer]
Hi @kwon-encored

Glad to have to interested in our work. The satellite prediction project (CloudCast) that we are working on with the Alan Turing Institute is early stage work so we don't really have any assets to share about it. There is a another article [here](https://manchesterprize.org/quartz-solar-ai-nowcasting) about the work, but it doesn't cover any more technical details. 

You asked about the approach of this project and how it compares to PVNet. Actually this new project will be integrated with PVNet. The basic approach is to train an ML model to forecast future satellite images out to a few hours ahead. So we are esentially creating an AI weather model, but instead of forecasting temperature, wind speed, and precipitation etc, we are forecasting multispectral satellite images.

We will then ingest those satellite predictions into PVNet in the same way that we ingest other NWPs like ECMWF or MetOffice.

We hope to write this us a paper in 2025
